Australian Digital Observatory/assets/static/style_custom.css

```diff
@@ -59,28 +70,13 @@ h3, h4 {
 
 .accordion-header * {
```

Comment:
Does the font for the accordion need to be changed as well?

Reply:
Hi Alice, I've inspected the accordions and I believe it's the same font. I do understand why you might see the font as different from the rest of the body text. I've changed the boldness and size of the accordion text. Hopefully now they look more congruent, but let me know if you're still seeing a different font.